charts/monitoring/grafana/dashboards/mla/mla-components.json

```diff
@@ -0,0 +1,2022 @@
+{
+  "annotations": {
+    "list": []
+  },
+  "description": "Mla Deployments/Statefulset Overview",
```

Comment:
Capital `MLA` or even better "User Cluster MLA" - to use the same naming as in the docs: [User Cluster MLA](https://docs.kubermatic.com/kubermatic/v2.18/guides/monitoring_logging_alerting/user_cluster/)

Reply:
Fixed the description.